app/Http/Controllers/SubpageController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Subpage;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class SubpageController extends Controller
+{
+    /**
+     * Show a subpage from an id.
+     *
+     * @param int $id
+     * @return View
+     */
+    public function show($id) : View
+    {
+        $subpage = Subpage::public()
+            ->findOrFail((int) $id);
+        return view('subpage.show', ['subpage' => $subpage]);
+    }
+
+    /**
+     * Show a subpage from a path.
+     *
+     * @param string$path
+     * @return View
+     */
+    public function showFromPath(string $path) : View
```

Comment:
@abm-devmonk-purple Thoughts on moving this to it's own invokable controller?
Route:
`Route::get('{path}', 'SubpagePathController')->name('subpage.path');`
Controller:
`public function __invoke(string $path) : View { ... }`

This way we can stick to the resourceful controllers/methods only -> fallback to invoke/single-action.

Reply:
Done in e6ac5b1d5fd457b5616405fefd32d856b9624c3a.